superset/assets/src/components/StackTraceMessage.jsx

```diff
@@ -4,12 +4,15 @@ import PropTypes from 'prop-types';
 import { Alert, Collapse } from 'react-bootstrap';
 
 const propTypes = {
-  message: PropTypes.string,
-  queryResponse: PropTypes.object,
+  message: PropTypes.node.isRequired,
+  link: PropTypes.string,
+  stacktrace: PropTypes.string,
```

Comment:
camelcase `stackTrace` for consistency with `StackTraceMessage` in the class name and `showStackTrace`.